src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -275,4 +280,15 @@ private function addFlashMessageProductV2IsDisabled(): void
             )
         );
     }
+
+    private function addFlashMessageNotMultistoreCompatible(): void
+    {
+        $this->addFlash(
+            'error',
+            $this->trans(
+                'The new experimental product page is disabled when multistore is used, because multistore is not properly implemented yet.',
```

Comment:
@Julievrz I couldn't use the message suggested in the issue because it didn't really suit the situation... The page is disabled when multistore is used, whatever context we are in...

About the wording I used, I let you find a better way to say it, it's just so that you get an idea of what this is about 👍

Reply:
'This experimental page is not yet compatible with the multistore feature. To access the page, please [1]disable the multistore feature[/1]' in Admin.Notifications.Info

Thanks! @matthieu-rolland